src/io/flutter/utils/ColorIconMaker.java

```diff
@@ -26,15 +27,15 @@ public void paintIcon(Component c, Graphics g, int x, int y) {
             GraphicsUtil.setupAAPainting(g2);
             // draw a black and gray grid to use as the background to disambiguate
             // opaque colors from translucent colors.
-            g2.setColor(Color.white);
-            g2.fillRect(iconMargin, iconMargin, getIconWidth() - iconMargin * 2, getIconHeight() - iconMargin * 2);
-            g2.setColor(Color.gray);
-            g2.fillRect(iconMargin, iconMargin, getIconWidth() / 2 - iconMargin, getIconHeight() / 2 - iconMargin);
-            g2.fillRect(getIconWidth() / 2, getIconHeight() / 2, getIconWidth() / 2 - iconMargin, getIconHeight() / 2 - iconMargin);
+            g2.setColor(JBColor.white);
+            g2.fillRect(x + iconMargin, y + iconMargin, getIconWidth() - iconMargin * 2, getIconHeight() - iconMargin * 2);
```

Comment:
this logic was just plain wrong as I didn't understand how the paint method for an icon was supposed to work but it didn't matter before as I was drawing icons on the LHS of the window.